File: Samples/iOS/ObjC/NotionClientExample/ViewController.m
//
//  ViewController.m
//  NotionClientExample
//
//  Created by David De Bels on 15/05/2021.
//

#import <NotionClient/Notion.h>

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view.
    
    NotionClient *client = [NotionClient clientWithToken:@"NOTION_INTEGRATION_TOKEN"];
}

@end

File: Samples/iOS/Swift/NotionClientExample/ViewController.swift
//
//  ViewController.swift
//  NotionClientExample
//
//  Created by David De Bels on 15/05/2021.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view.
        
        let client = NotionClient.init(token: "NOTION_INTEGRATION_TOKEN")
    }


}

